Scoop/Controller/Home/HomeViewController.swift

```diff
@@ -35,7 +35,7 @@ class HomeViewController: UIViewController {
         setupTableView()
         setupPostRideButton()
         
-        getRides()
+//        getRides()
         //        setupSignOutButton()
```

Comment:
Can we document why these functions are commented out? Do we need them? If so, why? If not, can we delete them?

Reply:
I uncommented them after I updated the ride model and created a getAllRides request in the Network Manager. Will be sure to document that next time.